projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -352,6 +352,7 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
         const data = _.get(questionSetResponse, _.toLower(`result.${this.objectType}`));
         this.collectionTreeNodes.data.instructions = data.instructions ? data.instructions : '';
         this.collectionTreeNodes.data.outcomeDeclaration = data?.outcomeDeclaration;
+        
```

Comment:
@pavankumar0408 Please remove this unnecessary newline change it might be impacting the PR code coverage.